.github/workflows/ci.yml

```diff
@@ -32,13 +32,23 @@ jobs:
         with:
           distribution: corretto
           java-version: 11
+          cache: sbt
 
       - name: build + test
-        run: |
-          set -e
-          # Ensure we don't overwrite existing (Teamcity) builds.
-          LAST_TEAMCITY_BUILD=2360
-          export GITHUB_RUN_NUMBER=$(( $GITHUB_RUN_NUMBER + $LAST_TEAMCITY_BUILD ))
+        env:
+          # The GitHub runner has 7GB of RAM, lets give 4GB to Java.
+          # See https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources.
+          JAVA_OPTS: -Xmx4g
+        run: sbt clean scalafmtCheckAll scalafmtSbtCheck compile lib/test riffraff/test Universal/packageZipTarball
```

Comment:
Is there a particular reason that we need to run `lib/test riffraff/test` (rather than just `test`)?

Reply:
`sbt test` only runs the tests from the `riffraff` project. I'm not entirely sure why - it'll be something in the `build.sbt`, but can't quite see it.